packages/digit_scanner/lib/pages/qr_scanner.dart

```diff
@@ -421,22 +426,28 @@ class _DigitScannerPageState extends LocalizedState<DigitScannerPage> {
                         ),
                         footer: DigitElevatedButton(
                           onPressed: () async {
-                                  final bloc = context.read<DigitScannerBloc>();
-                                  codes.add(_resourceController.value.text);
-                                  bloc.add(
-                                    DigitScannerEvent.handleScanner(
-                                      barCode: state.barCodes,
-                                      qrCode: codes,
-                                    ),
-                                  );
-                                  if (widget.isGS1code &&
-                                      result.length < widget.quantity) {
-                                    DigitScannerUtils().buildDialog(
-                                      context,
-                                      localizations,
-                                    );
-                                  }
-                                },
+                            final bloc = context.read<DigitScannerBloc>();
+                            codes.add(_resourceController.value.text);
+                            bloc.add(
+                              DigitScannerEvent.handleScanner(
+                                barCode: state.barCodes,
+                                qrCode: codes,
+                              ),
+                            );
+                            if (widget.isGS1code &&
+                                result.length < widget.quantity) {
+                              DigitScannerUtils().buildDialog(
+                                context,
+                                localizations,
+                                widget.quantity,
+                              );
+                            }
+
+                            setState(() {
+                              manualCode = false;
+                              initializeCameras();
```

Comment:
why are we initializing Cameras while submit? It should be called in initstate right?

Reply:
This is when the manually entered code is enabled we are disabling the camera, So after disabling the manual code we have to again initialize the camera, else it will be unable to find the camera controller